card/mod/01_core/set/all/collection.rb

```diff
@@ -144,3 +144,47 @@ def set_search_params_variables! hash
   end
 end
 
+format :html do
+  view :tabs do |args|
+    tabs = ''
+    tab_panels = ''
+    card.item_cards.each_with_index do |item, index|
+      id = "#{card.cardname.safe_key}-#{item.cardname.safe_key}"
+      tabs += tab( id, item.name, index == 0 )
+      tab_content = nest item, item_args(args)
+      tab_panels += tab_panel( id, tab_content, index == 0 )
+    end
+
+    wrap_with :div, :role=>"tabpanel" do
+      [
+        content_tag(:ul, tabs.html_safe, :class=>"nav #{args[:tab_type]}", :role=>"tablist"),
+        content_tag(:div, tab_panels.html_safe, :class=>'tab-content')
+      ]
+    end
+  end
```

Comment:
this is cool.  Perhaps we could optimize and make it so hidden tabs load later with ajax (to be more efficient)